<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>501</width>
     <height>89</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="csvPathLabel">
        <property name="text">
         <string>csv文件路径:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="csvPathEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="csvPathButton">
        <property name="text">
         <string>浏览...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="binPathLabel">
        <property name="text">
         <string>bin文件路径:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="binPathEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="binPathButton">
        <property name="text">
         <string>浏览...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="odrPathLabel">
        <property name="text">
         <string>odr文件路径:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="odrPathEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="odrPathButton">
        <property name="text">
         <string>浏览...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>文件设置：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>网络选项：</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>561</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="netWaitLabel">
      <property name="text">
       <string>应答等待：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="netWaitBox"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="modelLabel">
      <property name="text">
       <string>调试模式：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="modelBox">
      <item>
       <property name="text">
        <string>远程模式</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>文件模式</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>即时模式</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="applyButton">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QCheckBox" name="msgLenLabel">
      <property name="text">
       <string>报文限长</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="msgLenBox"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>byte</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QCheckBox" name="interpolateTimeLabel">
      <property name="text">
       <string>插值时隙：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="interpolateTimeBox"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>196</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>调试设置：</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="tabDrcLabel">
      <property name="text">
       <string>面板方向：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="tabDrcBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>Vertical</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Horizontal</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="logFormatLabel">
      <property name="text">
       <string>日志格式：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="logFormatBox">
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
